modules/virt-removing-pci-device-from-cluster-cli.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-removing-pci-device-from-cluster_{context}"]
+= Removing PCI host devices from the cluster using the CLI
+
+To remove a PCI host device from the cluster, delete the information for that device from the `HyperConverged` custom resource (CR).
+
+
+.Procedure
+. Edit the `HyperConverged` CR in YAML format using the default editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv -o yaml
```

Comment:
We should drop `-o yaml` here, this shouldn't be required here.

Reply:
@kbidarkar Is the object configuration in YAML format by default? Is that why we don't need to specify `-o yaml`?

Reply:
Removed `-o yaml` from the command